src/utils/coordinateUtils.js

```diff
@@ -431,26 +432,27 @@ export const splitIfOutOfLV95Bounds = (coordinates) => {
 
 /**
  * @param {[Number, Number][]} coordinates
+ * @param {CoordinateSystemBounds} bounds
  * @param {[CoordinatesChunk]} previousChunks
  * @param {Boolean} isFirstChunk
  * @returns {[CoordinatesChunk]}
  */
-function splitIfOutOfLV95BoundsRecurse(coordinates, previousChunks = [], isFirstChunk = true) {
+function splitIfOutOfLV95BoundsRecurse(
+    coordinates,
+    bounds,
+    previousChunks = [],
+    isFirstChunk = true
+) {
```

Comment:
This function seems to use `TILEGRID_EXTENT_EPSG_3857` at line 475 in a `clip` function, not sure if here we also needs some adaptation ?

Reply:
It should actually use the tile grid extent of the correct projection for the CH bound

Reply:
I think we should maybe starts directly by having each projection system declares its extent, that would help readability and use cases. Instead of having `switch...case` clauses everywhere, I'll try going this way

Reply:
Split is now made on the base of the CoordinateSystemBounds, and is therefor "projection agnostic"